feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts

```diff
@@ -144,21 +145,14 @@ export class ConfiguratorAddToCartButtonComponent implements OnInit, OnDestroy {
       .extractRouterData()
       .pipe(
         switchMap((routerData) =>
-          this.configuratorCommonsService.getOrCreateConfiguration(
-            routerData.owner
-          )
-        ),
-        distinctUntilKeyChanged('configId'),
-        switchMap((configuration) =>
-          this.configuratorCommonsService.getConfigurationWithOverview(
-            configuration
-          )
+          this.configuratorCommonsService.getConfiguration(routerData.owner)
         ),
         filter((configuration) => configuration.overview != null),
         take(1),
         delay(0) //we need to consider the re-rendering of the page
       )
       .subscribe(() => {
+        this.focusService.clear();
```

Comment:
if i remove this line - the tests still pass, so i assume an additional test and/or an additional assert is missing.

Reply:
Done.